script_templates/Node/node_template.gd

```diff
@@ -2,7 +2,7 @@
 # meta-default: true
 
 # @tool
-class_name
+class_name TemplateClass
```

Comment:
Were these actually showing up as editor errors for the template? Or is this unrelated.

Reply:
Not within godot itself, but they were showing up as errors with the godot vscode extension